core/domain/contribution_stats_services.py

```diff
@@ -0,0 +1,458 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Funtions to get stats Displayed in contributor Admin Dashboard.
```

Comment:
I believe this file will encompass more than just getters.

```suggestion
"""Service methods for operating on contributor dashboard user stats.
```

Reply:
Changed it to `Service methods for operating on contributor admin dashboard user stats.`. I think it is more accurate as we are using this file for admin dashboard stats.